HazelcastStarterTest.testHazelcastInstanceCompatibility_withStarterInstance

- Fails on `Hazelcast-3.x-IbmJDK8-fips-nightly`, `Hazelcast-3.x-sonar` and `Hazelcast-3.x-nightly`
- Starting from [this Build #45](http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-3.x-IbmJDK8-fips-nightly/45/)
- Seems from commit: 6531b52
- Error Message
```
java.lang.ClassNotFoundException: com.hazelcast.instance.impl.HazelcastInstanceProxy
```
- Stack trace
```
com.hazelcast.test.starter.GuardianException: java.lang.ClassNotFoundException: com.hazelcast.instance.impl.HazelcastInstanceProxy
	at com.hazelcast.test.starter.HazelcastStarterUtils.rethrowGuardianException(HazelcastStarterUtils.java:53)
	at com.hazelcast.test.starter.HazelcastStarter.getNode(HazelcastStarter.java:127)
	at com.hazelcast.instance.impl.TestUtil.getNode(TestUtil.java:74)
	at com.hazelcast.test.HazelcastTestSupport.getNode(HazelcastTestSupport.java:258)
	at com.hazelcast.test.HazelcastTestSupport.getNodeEngineImpl(HazelcastTestSupport.java:266)
	at com.hazelcast.test.starter.constructor.test.MergePolicyProviderConstructorTest.testConstructor(MergePolicyProviderConstructorTest.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:277)
	at java.lang.Thread.run(Thread.java:811)
Caused by: java.lang.ClassNotFoundException: com.hazelcast.instance.impl.HazelcastInstanceProxy
	at java.net.URLClassLoader.findClass(URLClassLoader.java:609)
	at com.hazelcast.test.starter.HazelcastAPIDelegatingClassloader.loadClass(HazelcastAPIDelegatingClassloader.java:118)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:853)
	at com.hazelcast.test.starter.HazelcastStarter.getHazelcastInstanceImpl(HazelcastStarter.java:162)
	at com.hazelcast.test.starter.HazelcastStarter.getNode(HazelcastStarter.java:120)
	... 16 more
```

--- a/hazelcast/src/test/java/com/hazelcast/test/starter/constructor/test/MergePolicyProviderConstructorTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/test/starter/constructor/test/MergePolicyProviderConstructorTest.java
@@ -29,6 +29,7 @@ import com.hazelcast.test.starter.HazelcastStarter;
 import com.hazelcast.test.starter.constructor.MergePolicyProviderConstructor;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
@@ -40,6 +41,7 @@ import static org.junit.Assert.fail;
 
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(SlowTest.class)
+@Ignore("To be enabled in 4.1 with 4.x instances - see https://github.com/hazelcast/hazelcast/issues/15263")
 public class MergePolicyProviderConstructorTest extends HazelcastTestSupport {
 
     private HazelcastInstance hz;

--- a/hazelcast/src/test/java/com/hazelcast/test/starter/test/HazelcastStarterTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/test/starter/test/HazelcastStarterTest.java
@@ -48,6 +48,7 @@ import static org.junit.Assert.fail;
 
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(SlowTest.class)
+@Ignore("To be enabled in 4.1 with 4.x instances - see https://github.com/hazelcast/hazelcast/issues/15263")
 public class HazelcastStarterTest {
 
     private HazelcastInstance hz;

